meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-mender/mender-client/mender-client_%.bbappend

```diff
@@ -1,21 +1,21 @@
 EXTRADEPS = ""
 EXTRADEPS:tegra = "tegra-bup-payload tegra-boot-tools tegra-boot-tools-nvbootctrl tegra-boot-tools-lateboot${@' libubootenv-fake' if d.getVar('PREFERRED_PROVIDER_virtual/bootloader').startswith('cboot') else ''}"
 EXTRADEPS:tegra210 = "tegra-bup-payload tegra-boot-tools"
-EXTRADEPS:jetson-agx-orin-devkit = "tegra-bup-payload libubootenv-fake mender-update-verifier"
+EXTRADEPS:tegra194 = "tegra-bup-payload tegra-uefi-capsules libubootenv-fake mender-update-verifier "
+EXTRADEPS:tegra234 = "tegra-bup-payload tegra-uefi-capsules libubootenv-fake mender-update-verifier"
```

Comment:
Once we drop tegra210 I think these can all be just incorporated in EXTRADEPS:tegra since they aren't MACHINE specific.